src/node_main_instance.cc

```diff
@@ -84,6 +88,17 @@ NodeMainInstance::NodeMainInstance(const SnapshotData* snapshot_data,
   SetIsolateCreateParamsForNode(isolate_params_.get());
   Isolate::Initialize(isolate_, *isolate_params_);
 
+  cpp_heap_ = CppHeap::Create(
+      platform,
+      CppHeapCreateParams {
+          {},
+          WrapperDescriptor(
+              BaseObject::kEmbedderType,
+              BaseObject::kSlot,
+              0x90df
```

Comment:
A recent commit made `kEmbedderType` and `kSlot` the correct values here (https://github.com/nodejs/node/commit/8b0e5b19bd0).  The embedder id in that commit was specified as `0x90de`.  When Chromium initializes the `CppHeap` its embedder id is `0x1` (`kEmbedderBlink`).  Pre-cppgc it used to be the case that for embedder tracing you had to specify the embedder id when you create objects, and it would need to match.  With `cppgc`, you just use `cppgc::MakeGarbageCollected<>` and I believe it uses the values that were specified when the `CppHeap` was created.  As a result, add-ons using `cppgc` don't need to be aware of the value chosen by Node.  Given though that the reason `0x90de` was picked was so that non-`cppgc` objects don't get mixed up with `cppgc` objects, we probably want to use a different embedder id than `0x90de` which Node is currently using for non-`cppgc` objects.  I just picked the next number here.

Reply:
I've rebased this on top of https://github.com/nodejs/node/pull/48660 so we do not need to worry about the exact values of the IDs anymore.